internal/config/funcs/doc.go

```diff
@@ -0,0 +1,76 @@
+package funcs
+
+func Docs() map[string]string {
+	return map[string]string{
+		"abs":             "return the absolute value of the given number",
```

Comment:
I originally wanted to extract these from godocs but I see how thats very difficult, so maybe a more structured approach like this is better.

Reply:
This was just a fast and dirty first approach, so the first cut on the website had SOMETHING for the each function.